your.README.md

```diff
@@ -0,0 +1,2 @@
+Will put more information here later.  This will be the readme where
```

Comment:
If we're going to set this up for forking, probably people are going to start doing this right away, so shouldn't we put the info we want them to have in their README here before merging? 

We will at least want markdown-format header (title) and subheaders and dummy text so they can easily see how to do those.

Obvious things to have authors put here:
- Appropriate markdown format header/title
- Brief explanation of what their focus is in this repository/work
- List of authors with contributions (to be updated as they go)
- List of acknowledgments with contributions (to be updated as they go)

Things we should probably put here:
- Very brief summary of this model of paper writing (link to "paper writing as code development?)
- Encouragement to community to use Issues button to make suggestions, for discussion, feedback, etc.
- Note that specific changes can be proposed via pull requests (eventually this would have a link to info on how to use Git/GitHub for this) 
- Maybe link to LiveCoMS, one or two sentence summary of what it is and how this repo (their repo) relates? 
- List of released versions saying "none yet"
- Changelog ("none yet")

Reply:
Are you imaging a changelog that get automatically updated somehow, or we expect them to provide a summary of changes?

Reply:
@mrshirts - never seen a useful way to automatically update a changelog, [because of this](https://xkcd.com/1296/) , but if you have one I'd love to hear it. No, this would be the responsibility of a maintainer who is interested in keeping people up to date on what is happening. Presumably if they don't want to do it they should remove it.

Reply:
OK, just checking.  I added a pass at this to the pull request when you have time to review.